test/android.vcf

```diff
@@ -1,18 +1,18 @@
 BEGIN:VCARD
 VERSION:2.1
-N:Test;Cozy;;;
-FN:Cozy Test
+N;CHARSET=UTF-8;ENCODING=QUOTED-PRINTABLE:=64=65=20=43=6F=7A=79;=54=65=73=74=C3=A9;;;

```

Comment:
could you add another .vcf style instead (to be sure it still works with a non-quoted value ?